### ISSUE TITLE
[Relay] connections visible in the schema but raise runtime error

### ISSUE THREAD

**Issue by tsnobip** [hasura/graphql-engine#5218]
I have the impression you need to enable aggregations to have access to the Relay connections of a table but schema introspection would make the connections visible anyway for all the tables/views that reference it, which creates a runtime error:
```json
{
  "errors": [
    {
      "extensions": {
        "path": "$.selectionSet.node",
        "code": "unexpected"
      },
      "message": "could not lookup \"foo_connection\" in 'bar_table'"
    }
  ]
}
```

First I think it would be good to document the fact you need to enable aggregations to access connections, and secondly the fact that the connection looks available in the schema but is not fetchable looks like a bug to me.

I tested this on 1.3.0-beta.2 and 1.3.0-beta.3.

**Comment by rakeshkky** [comment]
@tsnobip Please give us the minimal reproduction setup. 
Thank you.

**Comment by tsnobip** [comment]
@rakeshkky here is a very simple repro setup:
https://github.com/tsnobip/hasura_gqle_5218
just 2 tables with 2 - 3 fields each and 1 connection.

**Comment by rakeshkky** [comment]
@tsnobip 
I'm able to fix the bug in the PR #5257. The connection fields are exposed regardless of allow aggregation check in the select permission. Thank you :smile: .

**Comment by tsnobip** [comment]
Wow that was fast, thank you very much :)
